lib/trivial.nix

```diff
@@ -1,17 +1,44 @@
 rec {
 
-  # Identity function.
+  /* The identity function
+     For when you need a function that does “nothing”.
+
+     Type: id :: a -> a
+  */
   id = x: x;
 
-  # Constant function.
+  /* The constant function
+     Ignores the second argument.
+     Or: Construct a function that always returns a static value.
+
+     Type: const :: a -> b -> a
```

Comment:
That’s implicit in that, like in `id` as well.

Since you only have an `a` (on which no operations are defined), you can only return it untouched (because there’s nothing you can do with it).